docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/tracker-setup/initialization-options/index.md

```diff
@@ -302,7 +302,9 @@ The POST path that is used to send POST requests to a collector can be change wi
 
 `postPath` defaults to the standard path: `/com.snowplowanalytics.snowplow/tp2`
 
-Note: changing `postPath` is non-standard behavior that does not conform to the default collector protocol. The collector configuration must be updated to support this path.
+Note: changing `postPath` is non-standard behavior that does not conform to the default collector protocol.
```

Comment:
Could you make this a `:::caution`? (https://docusaurus.io/docs/markdown-features/admonitions)

Reply:
(Or maybe just a `:::note`)